### ISSUE TITLE
program blocked for getProgramAccounts

### ISSUE THREAD

**Issue by cxp-13** [metaplex-foundation/mpl-token-metadata#147]
using `https://devnet.helius-rpc.com/?api-key=xxxxx` as RPC provider.
https://developers.metaplex.com/token-metadata/fetch#fetch-all-by-creator  
I want to call the `fetchAllDigitalAssetByCreator` function of metaplex, but `program blocked for getProgramAccounts, if you need this program please contact helius support`
but `fetchDigitalAsset`  it is work.

**Comment by npomfret** [comment]
Same question.  Looks like a number of providers had dropped this method without providing an alternative.

Have you tried quiknode?  I think they still have support

**Comment by MarkSackerberg** [comment]
Yes, most rpc providers do not allow GPA calls on Token Metadata. Extrnode does allow it currently I think.

A different solution, which would have a way quicker response time than a GPA would be to use [DAS API](https://developers.metaplex.com/das-api), which is supported by most of the [rpc providers](https://developers.metaplex.com/rpc-providers). You would want to look into the [getAssetsByGroup](https://developers.metaplex.com/das-api/methods/get-asset-by-group) function.
